tests/integration/test_logcollector/test_only_future_events/test_only_future_events.py

```diff
@@ -60,77 +60,70 @@
 import pytest
 import wazuh_testing.logcollector as logcollector
 from wazuh_testing import global_parameters
-from wazuh_testing.tools import monitoring, file
-from wazuh_testing.tools.configuration import load_wazuh_configurations
-from wazuh_testing.tools.monitoring import LOG_COLLECTOR_DETECTOR_PREFIX, WINDOWS_AGENT_DETECTOR_PREFIX
 from wazuh_testing.tools.services import control_service
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX, WINDOWS_AGENT_PREFIX, \
+                                               GENERIC_CALLBACK_ERROR_COMMAND_MONITORING
+from wazuh_testing.tools.configuration import load_configuration_template, get_test_cases_data
+from wazuh_testing.modules.logcollector import event_monitor as evm
 
 
-if sys.platform == 'win32':
-    prefix = monitoring.WINDOWS_AGENT_DETECTOR_PREFIX
-else:
-    prefix = monitoring.LOG_COLLECTOR_DETECTOR_PREFIX
-
 # Marks
 pytestmark = [pytest.mark.tier(level=0)]
 
-# Configuration
-DAEMON_NAME = "wazuh-logcollector"
-test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-configurations_path = os.path.join(test_data_path, 'wazuh_only_future_events_conf.yaml')
+prefix = LOG_COLLECTOR_PREFIX
+
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+
+# ------------------------------------------------ TEST_ACCEPTED_VALUES ------------------------------------------------
```

Comment:
accepted values?

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3484/commits/c381044fd71b864462fea0dc8549ca5865dac7bd)